index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Profolio</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>Jiachun</h1>
+      <nav>
+        <img src="./img/bar-icon.png" />
+      </nav>
+    </header>
+
+    <section id="intro">
+      <h1>
+        Hey There. <br />
+        I'm Jiachun
+      </h1>
+      <h2>I'm a Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li>
+          <a href="#" target="_blank"><img src="./img/github.png" /></a>
+        </li>
+        <li>
+          <a href="#" target="_blank"
+            ><img src="./img/Icon - Linkedin.png"
+          /></a>
+        </li>
+        <li>
+          <a href="#" target="_blank"><img src="./img/yeah.png" /></a>
+        </li>
+        <li>
+          <a href="#" target="_blank"><img src="./img/twitter.png" /></a>
```

Comment:
- [ ] Great work working on the previous changes. However, you need to adjust the position o your background image to match the original design

![image](https://user-images.githubusercontent.com/98453979/227010925-54b81a42-6063-48de-84f0-47e9ffaeb8a5.png)

### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/227011094-5ab8a72e-47af-4920-b17a-4c728b43cf31.png)

Reply:
@frank1738 I can see the background image with a proper position from my side, I am not sure why it happens.